components/infobox/commons/infobox_player.lua

```diff
@@ -0,0 +1,227 @@
+local Class = require('Module:Class')
+local Cell = require('Module:Infobox/Cell')
+local Infobox = require('Module:Infobox')
+local Template = require('Module:Template')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+local Namespace = require('Module:Namespace')
+local Links = require('Module:Links')
+local Flags = require('Module:Flags')
+--local Lua = require('Module:Lua')--needed for the next line only
+--local AgeCalculation = (Lua.requireIfExists('Module:Infobox/Player/Age') or require('Module:Infobox player/age')).get
+
+local getArgs = require('Module:Arguments').getArgs
+
+local Player = Class.new()
+local Language = mw.language.new('en')
+local _LINK_VARIANT = 'player'
+
+function Player.run(frame)
+    return Player:createInfobox(frame)
+end
+
+function Player:createInfobox(frame)
+    local args = getArgs(frame)
+    self.frame = frame
+    self.pagename = mw.title.getCurrentTitle().text
+    self.name = args.id or self.pagename
+
+    if args.game == nil then
+        return error('Please provide a game!')
+    end
+
+    local infobox = Infobox:create(frame, args.game)
+
+    local earnings = Player:calculateEarnings(args)
+    Variables.varDefine('earnings', earnings)
+    if earnings == 0 then
+        earnings = nil
+    else
+        earnings = '$' .. Language:formatNum(earnings)
+    end
+    local birthDisplay, deathDisplay, birthday, deathday = Player:birthAndDeath(args)
+    local status = Player:getStatus(args)
+
+    infobox :name(Player:name(args))
```

Comment:
Player infoboxes need the team icon next to the name, so that functionality needs to be added

Reply:
ehm yeah not on sc, sc2, warcraft
there we need the faction icons
but as a default we can add it via the function i call here
will look into it in the evening when back on pc^^

Reply:
Okay may need to think about yet another overridable function then

Reply:
already is the case, just need to adjust it for the default to be with team icon